docker-compose.yml

```
@@ -49,6 +63,18 @@ services:
       - glados-net
     restart: always
 
+  glados_monitor_state:
+    command: "--database-url postgres://postgres:${GLADOS_POSTGRES_PASSWORD}@glados_postgres:5432/glados follow-head-state --provider-url ${GLADOS_PROVIDER_URL?Glados monitor provider URL required}"
+    image: portalnetwork/glados-monitor:latest
+    environment:
+      RUST_LOG: warn,glados_monitor=info
+    depends_on:
+      - glados_audit
+      - glados_postgres
+    networks:
+      - glados-net
+    restart: always
+
```

We don't support latest state yet, this should be set to import the state_roots from 0 to 1 million

```suggestion
  glados_monitor_state:
    command: "--database-url postgres://postgres:${GLADOS_POSTGRES_PASSWORD}@glados_postgres:5432/glados populate-state-roots-range --start-block-number 0 --end-block-number 1000001 --provider-url ${GLADOS_PROVIDER_URL?Glados monitor provider URL required}"
    image: portalnetwork/glados-monitor:latest
    environment:
      RUST_LOG: warn,glados_monitor=info
    depends_on:
      - glados_audit
      - glados_postgres
    networks:
      - glados-net
    restart: always
```

Without this change the audits will just fail. We will enable `follow-head-state` after we have bridges gossiping latest